### PR TITLE
upgrade AdminLTE to version 2.3.3

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by OskarStark** [comment]
I'm not sure if this should be backported to `2.x`

//cc @sonata-project/contributors

**Comment by Soullivaneuh** [comment]
> I'm not sure if this should be backported to `2.x`

Absolutly not. :-)

**Comment by Soullivaneuh** [comment]
@OskarStark Did you manually test the upgrade?

**Comment by OskarStark** [comment]
> @OskarStark Did you manually test the upgrade?

Yes I do, i clicked through my whole project and it looks good 👍

**Comment by Soullivaneuh** [comment]
Please fill your PR of missing parts indicated by the "Needs" labels.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [3.0.0](https://github.com/olivierlacan/keep-a-changelog/compare/2.3.10...3.0.0) - 2016-05-08
 ### Added
+- Updated AdminLTE theme to version 2.3.3
```
Comment (OskarStark) [inline CHANGELOG.md]: is this the right section @Soullivaneuh ?

Comment (OskarStark) [inline CHANGELOG.md]: oh i think it needs to go to the "Changed" section, right?

**Comment by OskarStark** [comment]
done @Soullivaneuh

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Add the ``getMaxPageLinks``, ``setMaxPageLinks`` methods to the PagerInterface
 
 ### Changed
+- Updated AdminLTE theme to version 2.3.3
```
Comment (Soullivaneuh) [inline CHANGELOG.md]: Wrong place. Has to be on the `Unreleased` part.

Comment (OskarStark) [inline CHANGELOG.md]: ok ;-)

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Added
 
+### Changed
+- Updated AdminLTE theme to version 2.3.3
+
 ## [3.0.0](https://github.com/olivierlacan/keep-a-changelog/compare/2.3.10...3.0.0) - 2016-05-08
```
Comment (OskarStark) [inline CHANGELOG.md]: not part of this PR but is this URL for compare correct? oO @Soullivaneuh

Comment (Soullivaneuh) [inline CHANGELOG.md]: Well, obviously not...

**Comment by Soullivaneuh** [comment]
@OskarStark Could you please rebase?

**Comment by OskarStark** [comment]
done @Soullivaneuh

**Comment by guilleferrer** [comment]
Hi, 
I would like to use AdminLTE with sonata-admin 2.3 . Is there any patch/pull request for it ?
Many thanks in advance!

**Comment by Soullivaneuh** [comment]
@guilleferrer Sonata admin 2.x is not maintained anymore.
